codeforamerica/vita-min
# FYST-195 No longer skipping data review step

## [FYST-195](https://codeforamerica.atlassian.net/browse/FYST-195)
## Is PM acceptance required?
- Yes - don't merge until JIRA issue is accepted!
## What was done?
- We had been skipping the data review step. As discussed with Tiffany at standup we no longer are
## How to test?
- Test on Heroku. After the data transfer step, you should see this
![image](https://github.com/user-attachments/assets/6cfc3923-7ac4-4aef-b63d-c06666633dc8)


[FYST-195]: https://codeforamerica.atlassian.net/browse/FYST-195?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
